app/models/car.rb

```diff
@@ -1,8 +1,6 @@
 class Car < ApplicationRecord
   belongs_to :user
+  has_one :reservation, dependent: :destroy
 
-  validates :model, presence: true
-  validates :brand, presence: true
```

Comment:
@bobb-Rob the relationship between car and reservation as show here matches the one in the requirements, congratulations 👍